modules/oe_content_event/config/install/field.field.node.oe_event.field_featured_media.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_featured_media
+    - media.type.av_portal_photo
+    - media.type.image
+    - node.type.oe_event
+id: node.oe_event.field_featured_media
+field_name: field_featured_media
+entity_type: node
+bundle: oe_event
+label: 'Featured media'
+description: ''
+required: false
```

Comment:
should be required.

Reply:
Done through constraint. By documentation this field is required only if any fields within Description fields group filled in.